RecoJets/JetProducers/python/ECF_cff.py

```diff
@@ -3,30 +3,30 @@
 from RecoJets.JetProducers.ECFAdder_cfi import ECFAdder
 
 ecf = ECFAdder.clone(
-             src = cms.InputTag("ak8CHSJets"),
+             src = cms.InputTag("ak8PFJetsPuppi"),
              ecftype = cms.string("ECF")
              )
 
 ecfCbeta1 = ECFAdder.clone(
-             src = cms.InputTag("ak8CHSJets"),
+             src = cms.InputTag("ak8PFJetsPuppi"),
              ecftype = cms.string("C")
              )
```

Comment:
here and below, just clone from the most similar variant already:
```python
ecfCbeta1 = ecf.clone( ecftype = "C" )
```
this is substantially easier to maintain and is also more compact